presto-hive/src/test/java/com/facebook/presto/hive/TestFileSystemCache.java

```diff
@@ -50,6 +51,36 @@ public void testFileSystemCache()
         assertSame(fs3, fs4);
     }
 
+    @Test
+    public void testDisableCache()
+            throws IOException
+    {
+        HiveClientConfig hiveClientConfig = new HiveClientConfig();
+        HdfsEnvironment environment =
+                new HdfsEnvironment(
+                        new HiveHdfsConfiguration(new HdfsConfigurationInitializer(hiveClientConfig, new MetastoreClientConfig()), ImmutableSet.of(), hiveClientConfig),
+                        new MetastoreClientConfig(),
+                        new ImpersonatingHdfsAuthentication(new SimpleHadoopAuthentication()));
+        String filePath = Resources.getResource("test_file_system_cache/id.parquet").getPath();
+        Path path = new Path(filePath);
```

Comment:
```suggestion
        Path path = new Path(Files.createTempFile("id-", ".parquet").toUri());
```

Could we simply use a temp file here?